packages/studio-plugin/src/http/HttpService.ts

```diff
@@ -0,0 +1,78 @@
+import crossFetch from "cross-fetch";
+import { QueryParams } from "./types/QueryParams";
+
+/**
+ * The basic structure of a response from the Management API.
+ */
+interface ApiResponse {
+  response: unknown;
+  meta: {
+    uuid: string;
+    errors: {
+      message: string;
+      code: number;
+      type: string;
+    }[];
+  };
+}
+
+/**
+ * HttpService is a wrapper around the native implementation of AJAX-related
+ * matters.
+ */
+export class HttpService {
+  /**
+   * Performs a GET request.
+   */
+  static async get(
```

Comment:
Maybe dumb question but how come this works but calling from the browser doesn't? Wouldn't the origin when called from the node process just be like... the user's IP address or something, which would still fail the cross-origin check?

Reply:
I think having the correct CORS headers is only enforced by the browser, because even if I do a `curl` with the exact headers copied from the browser request (including the origin), the request is successful. So, the node process is able to get the response because there's no browser to intercept it